recipes/libsigcpp/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
```

Comment:
this file is not required, as you are not using the generic folder `all`

Reply:
Use a generic folder